### ISSUE TITLE
Incompatible with webpack's ProfilingPlugin

### ISSUE THREAD

**Issue by nikolaik** [jantimon/html-webpack-plugin#1652]
## Current behaviour 💣
The webpack build crashes when used with the ProfilingPlugin

webpack plugin config:

```
plugins: [
  new HtmlWebpackPlugin(),
  new webpack.debug.ProfilingPlugin()
]
```

Ref: https://github.com/webpack/webpack/issues/12102#issuecomment-847104887

## Expected behaviour ☀️

Don't crash

### Reproduction Example 👾

Here is a link to a code sandbox https://codesandbox.io/s/html-webpack-plugin-with-profiler-0xibi?file=/webpack.config.js

**Comment by jantimon** [comment]
I have no idea why HtmlWebpackPlugin works in so many configurations but not with ProfilingPlugin..

Is ProfilingPlugin messing around with Webpack internals?

I would be very happy for a PR

**Comment by zqcccc** [comment]
I solved this problem by putting ProfilingPlugin in the first position of the plugins array.

**Comment by Taggotty** [comment]
> I solved this problem by putting ProfilingPlugin in the first position of the plugins array. version: html-webpack-plugin:5.50 webpack: 5.61.0

Did not work for me with the same versions. Any other ideas?

**Comment by arezzinskikh** [comment]
Hello there, 
I get a same problem, any ideas/solutions for solving?
Thanx

**Comment by MaximeCheramy** [comment]
I confirm that this bug is fixed with Webpack 5.70.0.

**Comment by nikolaik** [comment]
> I confirm that this bug is fixed with Webpack 5.70.0.

Nice, thanks for checking!
